3_javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,90 @@
+let fromCurrency = 'EUR'
+let toCurrency = 'DKK'
+let amount = ''
+
+function setupCurrencyCalculator(fromCurrency, toCurrency) {
+    console.log('fromCurrency', fromCurrency)
+    console.log('toCurrency', toCurrency)
+
+    const API = "https://open.er-api.com/v6/latest/"
+
+    fetch(`${API}${fromCurrency}`)
+        .then((response) =>
+            response.json()
+            //console.log(response);
+        ).then((data) => {
+            const exchangeRates = data.rates;
+
+            const currencyList = Object.keys(exchangeRates);
+
+            const fromSelectBoxDomNode = createSelectBox(
+                currencyList,
+                fromCurrency,
+                'fromCurrencySelectBox',
+                (e) => {
+                    fromCurrency = e.target.value;
```

Comment:
Well done creating a function for creating the select option to avoid repetitive code.